Packages/systems.speckle.speckle-unity/Editor/Components/SpeckleReceiverEditor.cs

```diff
@@ -21,7 +26,21 @@ public override async void OnInspectorGUI()
         {
             var speckleReceiver = (SpeckleReceiver)target;
 
-            DrawDefaultInspector();
+            //Selection
+            bool isFrontend2 = speckleReceiver.Account.Selected?.serverInfo.frontend2 ?? true;
+            EditorGUILayout.PropertyField(_accountSelection);
+            EditorGUILayout.PropertyField(
+                _streamSelection,
+                new GUIContent(isFrontend2 ? "Project" : "Stream")
```

Comment:
why we still need this switch? shouldn't we just use "Project"

Reply:
Good question,

This allows FE1 users to still see "Stream", "Branch", and "Commit" in the main UI, And FE2 users to see "Project", "Model", and  "Version".

I didn't intend to add it, but seeing as it was only only a handful lines, I decided to experiment.
I think its easy enough, might help avoid any confusion as users make the transition, and something we could kill in a few releases anyway.

What do you think? @bimgeek @oguzhankoral @clairekuang

Reply:
That was the case also in excel and I've changed it on my PR today, so users will always see new terms even if they have old account. I though this was a decision we made.

Reply:
I considered preserving this behavior for DUI2 as well, but the decision there is to just remove all old terminology. I think it's best to align and not show old terminology regardless of account

Reply:
Ok, I'll remove this switching.